index.html

```diff
@@ -0,0 +1,328 @@
+    Nihil odio eum mollitia pariatur velit impedit consequuntur, eius molestiae quod quidem! Doloremque enim molestias corporis aliquid odio doloribus dolores labore quasi necessitatibus, sunt obcaecati rerum nam, accusantium temporibus beatae!
+    Commodi explicabo illo animi dolorem rerum provident itaque consectetur incidunt quia, voluptatem voluptas quo voluptate corrupti debitis ullam! Dolores tempora optio atque esse beatae ab. Neque dignissimos accusantium quasi ducimus?
+    Nemo cupiditate enim autem eveniet nulla! Voluptatem fugit sint quibusdam eaque nulla, explicabo fuga at dolor voluptate ipsam nam earum ipsa unde placeat aperiam? Laboriosam nulla veritatis eligendi ad similique.
+    Beatae doloremque tempore aliquid recusandae aut. Reiciendis autem magnam et ea minima voluptates, minus, perspiciatis nobis esse eaque porro repudiandae accusamus? Sequi incidunt soluta nostrum, commodi vitae doloremque reprehenderit autem.
+    Rem repellendus ratione iste molestias nesciunt ex ipsa quae accusamus, ut obcaecati quisquam. Veniam sequi error ipsam, libero pariatur saepe deserunt quas nemo explicabo numquam deleniti quibusdam ex hic odit.
+    Aperiam similique eveniet labore omnis quam. Blanditiis error unde nisi architecto quisquam ut, voluptatibus, a sit exercitationem natus saepe quibusdam eveniet quod aliquid perspiciatis harum iste cum officiis, inventore delectus!
+    Nisi libero excepturi in debitis ex quisquam omnis iure? Doloribus id voluptatum suscipit, ut quidem unde praesentium incidunt ad? Eius ad et, facilis iusto recusandae ex minus unde commodi temporibus.
+    Atque mollitia sapiente fugiat accusamus alias minus eaque magnam voluptatem repellat provident delectus, deleniti dolores modi consequatur a nam, adipisci, sint exercitationem suscipit dolore numquam iste perferendis itaque? Ut, reiciendis?
+    Rem tempore fugiat, consequuntur sit repudiandae aperiam culpa suscipit odit impedit aliquid quis earum temporibus quae molestiae ratione! Distinctio excepturi, ratione illum est tempore ut at adipisci illo recusandae quam!
+    Optio numquam odio provident ipsam hic, at quo et? Quam, similique animi. Incidunt, doloremque sunt provident nostrum qui dolore cumque dicta eveniet modi. Labore ratione doloremque doloribus adipisci aliquam eaque!
+    Odit optio at quis fuga vel sed in molestiae, ipsum, aut doloremque tempore vitae. Culpa provident, quod ratione sint numquam id, vero natus et, sed libero at. Adipisci, quod ducimus.
+    Ullam reprehenderit natus perferendis unde quas hic sunt rem explicabo beatae molestiae voluptates, tenetur exercitationem facilis, pariatur non optio assumenda autem facere ex? Aut dolorum praesentium cumque numquam. Dolorum, a!</p>
+    <section id="footer" class="container p-3">
```

Comment:
- [ ] Please use the more semantic `footer` tag in here instead of `section`.

- [ ] The footer logo's size is not matching the origin one, too big on wide screens and too small on tablet and mobile, the list links are wrapping more than expected compared to their original behavior, and the text color is wrong.
Compare your footer and the original one below, and also in your browser in the responsive mode, and make sure it's closer to the original:
![Screenshot_2021-01-25 Newsweek(7)](https://user-images.githubusercontent.com/16193212/105739107-84640a00-5f16-11eb-9218-701511414fdd.png)
![Screenshot_2021-01-25 NewsWeek(6)](https://user-images.githubusercontent.com/16193212/105739113-84fca080-5f16-11eb-9a6f-5a00c1c8c267.png)